authentication-demo-service/src/main/java/io/mosip/authentication/demo/service/helper/KeyMgrUtil.java

```diff
@@ -391,9 +391,14 @@ public boolean updatePartnerCertificate(String partnerType, X509Certificate upda
     public String getKeysDirPath(String certsDir, String moduleName) {
       	String domain = environment.getProperty(DOMAIN_URL, "localhost").replace("https://", "").replace("http://", "").replace("/", "");
   		
-          // Default to temp folder in the contianer where Authdemo service is running
+          // Default to temp folder in the container and also windows where Authdemo service is running
       	String certsTargetDir = System.getProperty("java.io.tmpdir")+ File.separator + "AUTHCERTS";
       	
+      	if (System.getProperty("os.name").toLowerCase().contains("windows") == false) {
+      		// if OS is non-windows override certsTargetDir with directory which works on docker
+      		certsTargetDir = "/home/mosip/authcerts";
```

Comment:
Please take this from property instead of hardcoding.

Reply:
We already enhanced auth demo service to take certDir as a parameter to the API call. The default value is to address. If the parameter is not passed and as per discussion with DevOps. for now they're going to fix this default directory.